test/fixtures/es-class-assign-property-variable/expected-wrap-es5.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
-var propTypes = {
+var propTypes = process.env.NODE_ENV !== "production" ? {
   foo: PropTypes.string
```

Comment:
Can you add some tests for `get bar() { return PropTypes.string; }` and similar?

Reply:
Also, it'd be good to ensure that this doesn't crash:
```js
var propTypes = {
  ...({ foo: PropTypes.string }),
};
```